test/Experimental/MatrixGroups-test.jl

```diff
@@ -0,0 +1,12 @@
+@testset "GAP matrix groups" begin
+
+    m = matrix(ZZ, [0 1 ; -1 0])
+    gapM = Oscar.MatrixGroups._wrap_for_gap(m)
+    G = GAP.Globals.Group(gapM)
+    GAP.Globals.SetNiceMorphismForJuliaMatrixRepGroup(G)
+    @test GAP.Globals.Size(G) == 4
+    
+    m2 = matrix(QQ, [ -1 0; 0 1]);
+    G = Oscar.MatrixGroups.MatrixGroup([m,m2])
```

Comment:
Actually I think this line should give an error (which BTW should be tested using `@test_throws`). Reason: the two matrices have different parents / base rings. I think we should reject that with a suitable error.

Also, a test using an actual number field would be nice. Something like this:
```julia
K, a = quadratic_field(-1)
m1 = matrix(K, [ a 0 ; 0 a ])
m2 = matrix(K, [ 0 1 ; -1 0 ])
```
and then test with those